src/FxKit/Option/Option.NaturalTransformations.cs

```diff
@@ -9,13 +9,14 @@ namespace FxKit;
 public static partial class Option
 {
     /// <summary>
-    ///     Maps the <see cref="Option{T}" /> to a <see cref="Result{T,E}" />.
+    /// Returns a result containing the value from an <see cref="Option{T}"/> if it exists.
+    /// Otherwise, returns a result with the specified error value.
     /// </summary>
-    /// <param name="source"></param>
-    /// <param name="selector"></param>
-    /// <typeparam name="T"></typeparam>
-    /// <typeparam name="R"></typeparam>
-    /// <returns></returns>
+    /// <typeparam name="T">The type of the value in the result.</typeparam>
+    /// <typeparam name="R">The type of the error in the result.</typeparam>
+    /// <param name="source">The input <see cref="Option{T}"/>.</param>
+    /// <param name="selector">A function that provides the default error value to return if the option is empty.</param>
```

Comment:
I don't think the `default` qualifier is needed here.

```suggestion
    /// <param name="selector">A function that provides the error value to return if the option is empty.</param>
```

Reply:
Done